src/frontend/src/catalog/table_catalog.rs

```diff
@@ -615,6 +625,7 @@ mod tests {
                 version: Some(TableVersion::new_initial_for_test(ColumnId::new(1))),
                 watermark_columns: FixedBitSet::with_capacity(2),
                 dist_key_in_pk: vec![],
+                cardinality: Cardinality::default(),
```

Comment:
nit: what about `unknown` instead of `default`?

Reply:
(btw, hope to have https://github.com/tokio-rs/prost/issues/334 😇

Reply:
> nit: what about `unknown` instead of `default`?

The `impl Default` is required for the cardinality visitor as the default behavior, but indeed we can add a `unknown` method to make it more explicit here.